lib-es5/api.js

```diff
@@ -111,20 +111,22 @@ exports.resource = function resource(public_id, callback) {
   resource_type = options.resource_type || "image";
   type = options.type || "upload";
   uri = ["resources", resource_type, type, public_id];
-  return call_api("get", uri, only(options, "exif", "colors", "derived_next_cursor", "faces", "image_metadata", "pages", "phash", "coordinates", "max_results"), callback, options);
+  return call_api("get", uri, only(options, "exif", "colors", "derived_next_cursor", "faces", "image_metadata", "pages", "phash", "coordinates", "max_results", "versions"), callback, options);
 };
 
 exports.restore = function restore(public_ids, callback) {
   var options = arguments.length > 2 && arguments[2] !== undefined ? arguments[2] : {};
 
+  options.content_type = 'json';
```

Comment:
@strausr , why is that needed?

Reply:
This is done to accept an array as part of the options. Meaning this is how we know to parse an array.

Reply:
Not sure I get what you mean

Reply:
By default, we send all our requests URL-encoded (even post requests).

It seems our servers are unable (bug, or by design) to parse arrays sent this way.

We don't have this problem once we change the payload to JSON.
We have this line of code littered throughout the project (setting the content_type).

We might want to investigate why we need this (it exists in other SDKs as well)